src/main/java/io/github/thebusybiscuit/exoticgarden/ExoticGarden.java

```diff
@@ -892,6 +898,22 @@ public static ItemStack harvestPlant(Block block) {
 		return null;
 	}
 
+	public static void harvestFruit(Block fruit) {
+		Location loc = fruit.getLocation();
+		SlimefunItem check = BlockStorage.check(loc);
+		if (check == null) {
+		    return;
+        }
+
+        if (treeFruits.containsValue(check.getID())) {
```

Comment:
Oh i see, you are only using it for a .contains() check anyway, in that case a HashSet would probably be most optimal.